lib/main.js

```diff
@@ -77,6 +74,14 @@ module.exports = class Client {
         });
     }
 
+    getHeaders () {
+        const headers = {};
+        if (this.serverId) {
+            headers['origin-server-id'] = this.serverId;
+        }
+        return headers;
```

Comment:
```js
return {
  ...{  'origin-server-id': this.serverID || undefined},
};
```

I think that removes it if it's not there. Not sure if clearer, though

Reply:
changed this around.